app/services/allocations/create_in_nomis.rb

```diff
@@ -0,0 +1,28 @@
+module Allocations
+  class CreateInNomis
+    def self.call(move)
+      booking_id = move.person.latest_nomis_booking_id
+
+      body = {
+        booking_id: booking_id,
+        body_params: {
+          fromPrisonLocation: move.from_location.nomis_agency_id,
+          toPrisonLocation: move.to_location.nomis_agency_id,
```

Comment:
if we don't have a to location, will that be ok with nomis (I know we have a type of move that allows this to be nil) but wondering what the implications are on nomis?

Reply:
Good catch. Will add coverage and a nil guard for this